packages/@aws-cdk/cx-api/test/fixtures/missing-context/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.10.0",
+  "version": "0.0.0",
```

Comment:
Shouldn't these be 99.99.99 (that's not the CDK version, right?)

Reply:
using 99.99.99 will fail the tests because it higher than the current schema version (`2.0.0`).